src/renderer/families/polkadot/Nomination/index.js

```diff
@@ -293,6 +280,7 @@ const Nomination = ({ account }: Props) => {
                 mr={2}
```

Comment:
Just noticed a non-related related bug.
This `mr` is not supposed to be here, nominate is not aligned correctly.

If you don't want to fix this in this PR, i can plan to fix this while doing a split of this file after your PR is merged.